google/cloud/bigquerycontrol/CMakeLists.txt

```diff
@@ -0,0 +1,215 @@
+# ~~~
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ~~~
+
+include(GoogleCloudCppLibrary)
+
+include(GoogleapisConfig)
+set(DOXYGEN_PROJECT_NAME "Cloud BigQuery Control API C++ Client")
+set(DOXYGEN_PROJECT_BRIEF
+    "A C++ Client Library for the Cloud BigQuery Control API")
+set(DOXYGEN_PROJECT_NUMBER "${PROJECT_VERSION}")
```

Comment:
comment: we used to add ` (Experimental)`...

https://github.com/googleapis/google-cloud-cpp/blob/d43e267e19db8e2d556d17ccb0f5649f3dd6beca/cmake/GoogleCloudCppLibrary.cmake#L175-L176

... but honestly, I don't think this field makes it to cloud.google.com/cpp. I think it is a relic.